client/driver.go

```diff
@@ -66,7 +66,8 @@ var (
 	globalSeqNo         uint64
 	randSource          = rand.New(rand.NewSource(time.Now().UnixNano()))
 
-	defaultConfigFile = "~/.cql/config.yaml"
+	// Default path of config file
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported var DefaultConfigFile should be of the form "DefaultConfigFile ..."